store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/operation/handler/impl/FederatedGetTraitsHandlerTest.java

```diff
@@ -287,6 +287,11 @@ protected void addAdditionalOperationHandlers() {
             return null;
         }
 
+        @Override
+        protected OutputOperationHandler<GetTraits, Set<StoreTrait>> getGetTraitsHandler() {
+            return null;
```

Comment:
This should maybe return an actual handler to retain the same functionality as `getTraits()`